docs/detections/prebuilt-rules/rule-details/suspicious-remote-registry-access-via-sebackupprivilege.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[suspicious-remote-registry-access-via-sebackupprivilege]]
+=== Suspicious Remote Registry Access via SeBackupPrivilege
+
+Identifies remote access to the registry via an account with Backup Operators group membership. This may indicate an attempt to exfiltrate credentials via dumping the SAM registry hive in preparation for credential access and privileges elevation.
```

Comment:
```suggestion
Identifies remote access to the registry using an account with Backup Operators group membership. This may indicate an attempt to exfiltrate credentials by dumping the Security Account Manager (SAM) registry hive in preparation for credential access and privileges elevation.
```